src/assets/styles/pikaday.scss

```diff
@@ -0,0 +1,247 @@
+/*!
```

Comment:
This seemed like the best way to include the pikaday styles to tweak them ourselves